code/modules/core_implant/cruciform/cruciform.dm

```diff
@@ -79,6 +81,12 @@ var/list/disciples = list()
 		remove_cyber()
 	if(wearer && wearer.stat == DEAD)
 		deactivate()
+	if(wearer && wearer.stats && wearer.stats.getPerk(/datum/perk/channeling) && round(world.time) % 5 == 0)
+		power_regen -= channeling_boost  // Removing the previous channeling boost since the number of disciples may have changed
+		wearer.visible_message(SPAN_DANGER("Old [channeling_boost]"))
```

Comment:
Having this here spams the bearer every few seconds due to it being in the processing tab, commenting it out removes the issue. It may be better to move the bonus to the status tab to make it checked like power reserves versus the infinite chat spam.